src/components/NavBar/MenuDropdown.css.ts

```diff
@@ -26,6 +26,7 @@ export const MenuRow = style([
   {
     lineHeight: '24px',
     textDecoration: 'none',
+    alignItems: 'flex-start',
```

Comment:
does this change on flag change? i noticed you changed it in the last pr

Reply:
shouldn't change anything for existing rows! Needed flex-start alignment when there's both a title & subtext like in the wallet app MenuRow though